package.json

```diff
@@ -518,6 +518,16 @@
       "src/react-native-shadow-stack/**/*.*"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,graphql}": [
+      "yarn lint:ci"
```

Comment:
i _think_ lint-staged is going to try and append the staged files to this command, but since our lint command
```
"lint:ci": "yarn format:check && yarn lint:ts && yarn lint:js --quiet",
```
appends a `.` to the command, it will override and lint all the files. 
```
eslint --cache .
```

we should instead create a new command without the `.` or maybe modify the existing one if there aren't any repercussions to doing so.

Does this make sense @BrodyHughes ? Might cut down on some pre-commit time.

Reply:
ohhh i see ty